content/en/serverless/step_functions/troubleshooting.md

```diff
@@ -40,8 +40,8 @@ If your organization has an existing all-encompassing index with a low limit, pl
 Please enable the `Include execution data` option on the state machine's logging. After enabling this option, log execution input, data passed between states, and execution output is logged. The Datadog backend uses the logs to construct these step spans for you.
 
 ## Some step spans are missing in the traces
-- For actions, we support basic actions of Lambda and DynamoDB. For example, Lambda Invoke, DynamoDB GetItem, DynamoDB PutItem, DynamoDB UpdateItem and more.
-- `Wait`, `Choice`, `Success`, `Fail`, and `Pass` are supported, while `Map` and `Parallel` are not. You are able to see parallel executing spans stacked on top of each other, but no `Parallel` spans show on the flame graph.
+- Actions of Lambda, DynamoDB, StepFunction, and most of the other AWS services are supported, but`Activity` has not been supported yet.
```

Comment:
```suggestion
- Actions from Lambda, DynamoDB, StepFunction, and most of the other AWS services are supported. `Activity` is not supported.
```
Can you give me more context about what `Activity` means in this context?

Reply:
@urseberry Here is the [AWS's doc about StepFunction activity](https://docs.aws.amazon.com/step-functions/latest/dg/concepts-activities.html).